includes/elements/section.php

```diff
@@ -837,7 +837,7 @@ protected function register_controls() {
 				'type' => Controls_Manager::DIMENSIONS,
 				'size_units' => [ 'px', '%' ],
 				'selectors' => [
-					'{{WRAPPER}}, {{WRAPPER}} > .elementor-background-overlay' => 'border-radius: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+					'{{WRAPPER}}, {{WRAPPER}} > :is(.elementor-background-overlay, .elementor-motion-effects-container)' => 'border-radius: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
```

Comment:
If you add that to both the section and column, maybe just add to the CSS something like this?

```CSS
.elementor-background-overlay {
    border-radius: inherit;
}
```

Reply:
i dont think its wise it creates dependency which might break in the future